### PR TITLE
fix: [computer] device display name

### PR DESCRIPTION
**Pull request by itsXuSt**
if idLable not empty for encrypted devices, show idLabel first

Log: as title

### CODE REVIEW

**Review by Johnson-zs — APPROVED** [review]

**Comment by xzl01** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/linuxdeepin/dde-file-manager/pull/1852#" title="Author self-approved">itsXuSt</a>*, *<a href="https://github.com/linuxdeepin/dde-file-manager/pull/1852#pullrequestreview-1926987848" title="Approved">Johnson-zs</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=linuxdeepin%2Fdde-file-manager).

<details open>
Needs approval from an approver in each of these files:

- **[debian/deepin/OWNERS](https://github.com/linuxdeepin/dde-file-manager/blob/master/debian/deepin/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
